test/e2e_node/density_test.go

```diff
@@ -217,34 +179,17 @@ var _ = framework.KubeDescribe("Density [Serial] [Slow]", func() {
 			itArg := testArg
 			It(fmt.Sprintf("latency/resource should be within limit when create %d pods with %d background pods",
 				itArg.podsNr, itArg.bgPodsNr), func() {
-				bgPods := newTestPods(itArg.bgPodsNr, ImageRegistry[pauseImage], "background_pod")
-				testPods := newTestPods(itArg.podsNr, ImageRegistry[pauseImage], podType)
 
 				createCadvisorPod(f)
-				rc := NewResourceCollector(monitoringInterval)
-
-				By("Creating a batch of background pods")
-				// creatBatch is synchronized
-				// all pods are running when it returns
-				f.PodClient().CreateBatch(bgPods)
-
-				time.Sleep(sleepBeforeCreatePods)
-
-				// starting resource monitoring
-				rc.Start()
+				rc := NewResourceCollector(containerStatsPollingPeriod)
 
-				// do a sequential creation of pod (back to back)
-				batchlag, e2eLags := createBatchPodSequential(f, testPods)
+				batchlag, e2eLags := runDensitySeqTest(f, rc, itArg)
 
-				rc.Stop()
-
-				// verify latency
 				By("Verifying latency")
-				verifyLatency(batchlag, e2eLags, itArg)
+				printAndVerifyLatency(batchlag, e2eLags, itArg, true)
 
-				// verify resource
 				By("Verifying resource")
-				verifyResource(f, itArg.cpuLimits, itArg.memLimits, rc)
+				printAndVerifyResource(f, rc, itArg.cpuLimits, itArg.memLimits, true)
```

Comment:
Why no benchmark test for sequence test? It seems that it's straight forward to add it. :)